utils/nodes.js

```diff
@@ -322,6 +322,9 @@ module.exports = {
     if (submitter_id) {
       metadata.data.submitter_id = submitter_id;
     }
+    if (metadata.data.consent_codes) {
+        metadata.data.consent_codes = consent_codes;
```

Comment:
Now that the data simulator is fixed to not generate random consent_codes, there is no need to overwrite the existing consent codes
So i think this should be
```
async submitGraphAndFileMetadata(..., consent_codes=null) {
  if (consent_codes) {
    // check that the dictionary has consent_codes
    if (!metadata.data.consent_codes) raise error
    metadata.data.consent_codes = consent_codes;
```